File: Sorting and Searching/distinct numbers/distinct.py
def distinct_numbers(n, *arr): 
    list2 = []
    for i in range(n): 
        if len(list2) == 0: 
            list2.append(arr[i])
            continue
        for j in range(len(list2)): 
            if list2[j] == arr[i]:
                break
            if j == len(list2) - 1:
                list2.append(arr[i])
    print(list2)
    return len(list2)

print(distinct_numbers(5, 2, 3, 2, 2, 3))

File: Sorting and Searching/distinct numbers/distinct.js
function distinctNumbers(size, ...arr) {
  const arr2 = [];
  for (let i = 0; i < size; i++) {
    if (arr2.length == 0) {
      arr2.push(arr[i]);
      continue;
    }
    for (let j = 0; j < arr2.length; j++) {
      if (arr2[j] == arr[i]) break;
      if (j == arr2.length - 1) arr2.push(arr[i]);
    }
  }
  console.log(arr2);
  return arr2.length;
}

console.log(distinctNumbers(5, 2, 3, 2, 2, 3));
